tiingo/api.py

```diff
@@ -88,7 +90,13 @@ def get_ticker_metadata(self, ticker):
         """
         url = "tiingo/daily/{}".format(ticker)
         response = self._request('GET', url)
-        return response.json()
+        data = response.json()
+        if fmt == 'json':
+            return data
+        elif fmt == 'object':
+            # inspired by https://stackoverflow.com/a/15882054
+            return json.loads(json.dumps(data),
```

Comment:
So this won't work as is because `data` is a list of dicts, so the list has no `keys()`. What we want to return is a list of `NamedTuples`.

I would recommend defining the `Ticker` class up near the top of the file, and then returning something like:

`[Ticker(*d) for d in data]`. 

The same issue applies for the `NewsItem` below, as the raw JSON response is a list, not a dict. 

All of this will be worth it, because the `NamedTuple` has a lower memory impact than native Dicts, leading to better performance. So thank you for working on this :)

Reply:
I don't understand, My cloned repo is up to date with the master. But still, travis build fails.

Is making a branch and working mandatory? I just cloned and started working. I am sorry, I am new to working with git. :-(